src/geefetch/data/get.py

```diff
@@ -130,7 +137,11 @@ def download_chip(
         else:
             log.debug(f"File {out} does not seem corrupted. Skipping download.")
             return out
-    data = data_get_lazy(**data_get_kwargs)
+    try:
+        data = data_get_lazy(**data_get_kwargs)
+    except ValueError as e:
+        log.error(f"ValueError in data_get_lazy for {out}: {e}")
+        raise DownloadError from e
```

Comment:
This will trigger a retry on ValueError during `data_get_lazy`. I would expect `data_get_lazy` to trigger when there is a misconfiguration or something else deterministic, not an error we would like to retry on. Am I missing something ?